Release: Gateway 2.8.4.8 (#7136)
Kong/docs.konghq.com: app/gateway/changelog.md

These changes are optimized to work better in systems with high throughput. 
In a low-throughput setting, these new settings may have visible effects in load balancing, where it can take more requests to start 
using all the upstreams than before.

### Fixes
#### Configuration

* Fixed an issue where an external plugin (Go, Javascript, or Python) would fail to
apply a change to the plugin config via the Admin API.
* Set the security level of gRPC's TLS to `0` when `ssl_cipher_suite` is set to `old`.
 
#### Core

* Updated the file permission of `kong.logrotate` to 644.
* Fixed the missing router section for the output of request debugging.
* Fixed a issue where the `/metrics` endpoint would throw an error when database was down.
* Fixed the UDP socket leak of the DNS module.

#### Plugins

* [LDAP Auth Advanced](/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`)
  * Fixed some cache-related issues which caused `groups_required` to return unexpected codes after a non-200 response.

* [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`)

>   * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
>   If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.

Reviewer: ```suggestion
  *  Fixed an issue where the `sync_rate` setting, when being used with the `redis` strategy did not properly revert to the `local` strategy if the Redis connection was interrupted with `sync_rate = 0`.
```

Author: Rephrased it a bit more and applied that.

>   * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
>   If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.

Author: ```suggestion
  * Fixed an issue where, if `sync_rate` was set to `0` and the `redis` strategy was in use, the plugin did not properly revert to the `local` strategy if the Redis connection was interrupted.
```

* Fixed the UDP socket leak of the DNS module.

#### Plugins

* [LDAP Auth Advanced](/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`)
  * Fixed some cache-related issues which caused `groups_required` to return unexpected codes after a non-200 response.

* [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`)
  * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
  If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.

* [Rate Limiting](/hub/kong-inc/rate-limiting/) (`rate-limiting`), [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`), [GraphQL Rate Limiting Advanced](/hub/kong-inc/graphql-rate-limiting-advanced/) (`graphql-rate-limiting-advanced`), and [Response Rate Limiting](/hub/kong-inc/response-ratelimiting/) (`response-ratelimiting`)
  * Fixed an issue where any plugins using the `rate-limiting` library, when used together, 
  would interfere with each other and fail to synchronize counter data to the central data store.

### Dependencies

> * Bumped `lua-resty-openssl` to 1.2.1

Reviewer 2: ```suggestion
```

This is removed. See https://github.com/Kong/kong-ee/pull/8638/files.
